logging/test/logging_integration_test.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:logging/logging.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sentry_logging/sentry_logging.dart';
+import 'package:sentry_logging/src/version.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+import 'mock_hub.dart';
+
+void main() {
```

Comment:
We could add a text if the logged level is equal to min, as we only have ones above or below.